docs/mkdocs.yml

```diff
@@ -60,7 +60,7 @@ nav:
   - Amazon EMR: https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-iceberg-use-cluster.html
   - Snowflake: https://docs.snowflake.com/en/user-guide/tables-iceberg
   - Impala: https://impala.apache.org/docs/build/html/topics/impala_iceberg.html
-  - Doris: https://doris.apache.org/docs/dev/lakehouse/multi-catalog/iceberg
+  - Doris: https://doris.apache.org/docs/dev/lakehouse/datalake-analytics/iceberg
```

Comment:
should this actually be https://doris.apache.org/docs/lakehouse/datalake-analytics/iceberg instead of having `dev` in the URL?

Reply:
"dev" refers to the documentation for the master branch of Apache Doris.